README.md

```diff
@@ -2,5 +2,5 @@
 
 Welcome to the BrainIAK tutorial repository. 
 
-**Coming Soon:** In the next few weeks we will be releasing a set of tutorials that cover topics from basics to advanced fMRI analysis.
+**Coming Soon:** In early 2019 we will be releasing a set of tutorials that cover topics from basics to advanced fMRI analysis.
```

Comment:
The early 2019 makes the message consistent with the other sites and posters.

Reply:
Sure.